### PR TITLE
Translation setup

### PR DESCRIPTION
**Pull request by eschaefer**
- Adds Middleman, which has a really straightforward way to [localize templates](https://middlemanapp.com/advanced/localization/)
- Build full static site, using `middleman build`
- Adds localized translations in `/locales/`
- Localized template based in `/source/localizable/`

Currently has 3 example locales:
- `en`
- `de`
- `ru`

To add Portuguese for example, put `pt.yml` in `/locales/`.

Menu items are automatically added to the page header, based on the `.yml` files present.

See test at http://temp.deadhare.com

### CODE REVIEW

File: locales/de.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+de:
```
Comment (eschaefer) [inline locales/de.yml]: @franklyandrea this would be the text to translate into German :)

**Comment by Tethik** [comment]
One minor problem I saw is that links to images etc have to be absolute now.
e.g. on
http://temp.deadhare.com/de/
Logo breaks.

Also the link to contact form is rendered as
http://temp.deadhare.com/de/de/contact/

**Comment by eschaefer** [comment]
@Tethik thanks for pointing those issues out! Fixed the relative paths.

**Comment by hacker-hacker** [comment]
Good job! One question and one comment:

1) Do we need to brush up (create in my case :) ) our Ruby skills, or is the Ruby part very small in this change?
2) The language links are quite prominent, especially on mobile. Maybe they can be made a little more neutral both in color and position?

**Comment by eschaefer** [comment]
@hacker-hacker Thanks! 

1. I am not a Ruby person at all, I just found Middleman really easy to work with. I think the Ruby syntax is a non-barrier for a site like this.
2. Definitely can shrink down the language links :)

File: locales/de.yml [hunk 2]
```diff
@@ -1,35 +1,29 @@
...
-    best place to start is to subscribe to our <a href=\"https://lists.securitywithoutborders.org/mailman/listinfo/swb-public\">public mailing list</a>."
-  footer: "For general inquiries, please send an email to %{address}.
-    For assistance requests, please use the more secure link at the top of the page."
+  follow_us: "Folg uns auf Twitter %{address}"
+  request_assistance: "Unterstützung Anfragen"
+  intro_paragraphs: "Im digitalen Zeiltalter sind diejenigen, die Korruption aufdecken, gegen Rassismus kämpfen 
+    und sich für Demokratie und Gerechtigkeit einsetzen, zunehmend Drohungen und Verfolgung im Netz ausgesetzt.
+    <br /><b>Security Without Borders ist ein offenes Kollektiv von HackernInnen und Cyber-Sicherheits ExpertenInnen</b>,
+    die JournalistInnen, MenschenrechtlerInnen und Non-Profit-Organisationen in Fragen der Cybersicherheit unterstützen."
+  activist_headline: "Bist du AktivistIn, JournalistIn oder MenschenrechtlerIn?
```
Comment (Tethik) [inline locales/de.yml]: Is this missing a `"` at the end?

File: config.rb [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+# Helpers
+###
+
+# Methods defined in the helpers block are available in templates
+helpers do
+  def home_url
+    "/#{I18n.locale}"
+  end
+  def contact_form_url
+    "//localhost:5000/#{I18n.locale}/contact/form"
```
Comment (eschaefer) [inline config.rb]: Replace this with url of the Flask form

**Comment by Tethik** [comment]
I would remove the russian localization until we actually have it in russian (right now it's the same english).

Also to run this on Fedora I had to install a javascript runtime.
In the gemfile:
gem "rb-inotify"
gem "therubyracer"

Then to run it I had to use `bundle exec middleman <cmd>`

Consider adding it to the readme.

**Comment by hacker-hacker** [comment]
Thanks for the great additions, @eschaefer! Now merged.

**Pull request merged**